src/pages/CoinPage.jsx

```diff
@@ -7,7 +7,7 @@ import { CryptoContext } from "../contexts/cryptoContext";
 import { PriceChangePercentageText } from "../components/PriceChangePercentageText";
 import { Arrow } from "../components/Arrow";
 import { CoinPagePlusInCircleIcon } from "../components/CoinPagePlusInCircleIcon";
-import { v4 as uuidv4 } from 'uuid';
+import { v4 as uuidv4 } from "uuid";
```

Comment:
Absolute imports should be at the top. Followed by components and finally utils and styles